src/plugin/home/Resources/styles/home.less

```diff
@@ -14,6 +14,54 @@
 
   margin: 0 floor((@grid-gutter-width / 2) - 5px);
 
+  .top-tabs {
+    display: flex;
+    padding-left: 0;
+    margin-bottom: 0;
+    list-style: none;
+  }
+
+  .top-item.dropdown {
+    position: relative;
+
+    .top-tab {
+      padding: @padding-large-vertical 25px @padding-large-vertical @padding-large-horizontal;
+
+      &::before {
+        display: inline-block;
+        margin-left: .255em;
+        vertical-align: .255em;
+        content: "";
+        border-top: .3em solid;
+        border-right: .3em solid transparent;
+        border-bottom: 0;
+        border-left: .3em solid transparent;
+        right: 10px;
+        top: 45%;
+        position: absolute;
+      }
```

Comment:
What is the goal of this block ?

Reply:
This creates the chevron in those top tabs that have a dropdown menu. I used a `before` because there is already an `after` attached to that element - it takes care for the blue pointer arrow.

Reply:
OK. You should add a FontAwesome icon intead : 
- When tab is active, the chevron will get the same color than the text and remain visible.
- You will be able to put a button on it to open/close the dropdown for touchscreen support.